### ISSUE TITLE
Dafault environment_variables should be accessible in initialize script

### ISSUE THREAD

**Issue by awattez** [DannyBen/bashly#265]
### Description
it seems that the environment variables defined in the global scope and which have a default value are not accessible within the `initialize.sh` script.

### Reproduction steps
Init an example project:
```sh
bashly init
bashly generate
```

Change the API_KEY environment variable to a default value `bashly.yml`:
```yaml
environment_variables:
- name: api_key
  help: Set your API key
  default: "AN_API_KEY_THAT_DOES_NOT_EXIST"
```

Add this line in the `initialize.sh` script:
```sh
echo "API_KEY has value $API_KEY"
```

Generate again:
```sh
bashly generate
```

Run the command line cli:
```sh
./cli
API_KEY has value
```
There is no display of the default value for the API_KEY environment variable

### Expected behavior

```sh
./cli
API_KEY has value AN_API_KEY_THAT_DOES_NOT_EXIST
```
There is display of the default value for the API_KEY environment variable.

### Actual behavior

Run the command line cli:
```
./cli
API_KEY has value
```
There is no display of the default value for the API_KEY environment variable

Probably because of the `cli` file, the environment variable export line should not be inside the `parse_requirements` :
```sh
# :command.parse_requirements
parse_requirements() {
...
...
  # :command.environment_variables_filter
  export API_KEY="${API_KEY:-AN_API_KEY_THAT_DOES_NOT_EXIST}"
```
`parse_requirements` is executed after `initialize`

**Comment by DannyBen** [comment]
Hmm... good point. Thanks for the detailed issue, I will try to make it so.

**Comment by DannyBen** [comment]
This is now implemented in the master branch, and will be released soon.

If you need this immediately:

- If you are using the docker version, you can use the `dannyben/bashly:edge` image instead of the `lastest`
- If you are using the Ruby version, you can install the gem from GitHub (let me know if you need help with that).

Thanks for opening this issue.

**Comment by DannyBen** [comment]
This is now implemented in bashly 0.8.7
